app/services/platform-apps/api/modules/scene-transitions.ts

```diff
@@ -109,6 +111,80 @@ export class SceneTransitionsModule extends Module {
     throw new Error('Not Implemented');
   }
 
+  /**
+   * Get a list of transitions
+   *
+   * This includes all transitions the user has set up in SLOBS.
+   * For transitions managed by this App use `getAppTransitions`.
+   *
+   * @see {getAppTransitions}
+   */
+  @apiMethod()
+  async getTransitions(_ctx: IApiContext): Promise<Array<ITransition>> {
```

Comment:
Rather than importing the interface and using it for the return value, you should redefine a compatible interface in this file.  It's essentially boilerplate but it serves a very important purpose:  A developer down the line may innocently make a change to `ITransition` in the transitions service, without realizing it will break many apps in the wild.  By having them be separate interfaces, when they become incompatible, TypeScript will complain and it will force the developer to write a compatibility layer to translate the data.

Reply:
Fixed